client/my-sites/importer/importing-pane.jsx

```diff
@@ -33,6 +33,10 @@ const sum = ( a, b ) => a + b;
  * }
  */
 export const calculateProgress = ( progress ) => {
+	if ( ! progress ) {
```

Comment:
Can you add a comment here to clarify why we need this?

Reply:
Done: https://github.com/Automattic/wp-calypso/pull/60481/commits/bf705eb72d326bcb9324bf578ae507f5b21bc0c2